pom.xml

```diff
@@ -33,17 +33,18 @@
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
-        <version>1.0.6</version>
+        <version>1.0.7</version>
         <relativePath/>
     </parent>
     
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.version>8</maven.compiler.version>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
You can remove source and target as we only need the version property after 11...

Reply:
Done. Thanks :)